src/nvim/types.h

```diff
@@ -13,4 +13,5 @@ typedef unsigned char char_u;
 // Can hold one decoded UTF-8 character.
 typedef uint32_t u8char_T;
 
+typedef struct expand expand_T;
```

Comment:
We would prefer to make the tests find/define/import what they need, rather than adding imports or workarounds in the C source for the benefit of tests.